tslint.json

```diff
@@ -2,5 +2,8 @@
   "extends": ["tslint-config-stoplight"],
   "linterOptions": {
     "exclude": ["**/*.json"]
+  },
+  "rules": {
+    "prettier": "error"
```

Comment:
This makes sure we don't use the config enforced by tslint-config-stoplight, but choose the one we have in prettierc instead.